lib/active_reporting/report.rb

```diff
@@ -84,8 +85,10 @@ def select_aggregate
       end
     end
 
-    def dimension_joins
-      @dimensions.select { |d| d.type == :standard }.map { |d| d.name.to_sym }
+    def dimension_joins(join_method = ReportingDimension::JOIN_METHODS[:joins])
```

Comment:
I'm not necessarily a fan having constants from outside classes being default arguments for methods in other classes. There appears to be only two places where `dimension_joins` is called (both in this file). Could you do something similar to what you have on line 56 and call this with the normal "join" method name?

Reply:
Done